HashRegistrarSimplified.sol

```diff
@@ -266,17 +336,28 @@ contract Registrar {
         }
     }
     
+    /**
+     * @dev Cancel a bid you've submitted
```

Comment:
True. What is preventing someone from seeing another bidders sealed bid and cancelling it?